calibration/ADCPAM/CGINS-ADCPAM-55979__20200709.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+655979,CC_scale_factor1,0.61,
+655979,CC_scale_factor2,0.61,
+655979,CC_scale_factor3,0.61,
+655979,CC_scale_factor4,0.61,
```

Comment:
I think the cal date on the csv filename (and the vendor doc) should be 2020-06-24, which is the date on the Configuration Summary & Cert of Conformance in the vendor docs.